react/searchResult.css

```diff
@@ -237,6 +240,17 @@
 
   .totalProducts {
     justify-self: stretch;
+    padding-top: 4.5rem;
+  }
+
+  .searchOptionsBar{
+    display: grid;
+    display: -ms-grid;
+    -ms-grid-columns: 2fr 1px 1.5fr 1px 0.2fr;
+    grid-template-columns: 2fr 1px 1.5fr 1px 0.2fr;
+    grid-template-areas:
+      "orderby border filters borderseq switch";
+    width: 96%
```

Comment:
Why not 100%? Its akward to see the shadows of elements that are behind the bar (like the shadow of the footer)